src/test/java/com/google/devtools/build/lib/packages/TypeTest.java

```diff
@@ -372,6 +372,8 @@ public void testStringListDict() throws Exception {
     assertThat(converted).isEqualTo(expected);
     assertThat(converted).isNotSameInstanceAs(expected);
     assertThat(collectLabels(Types.STRING_LIST_DICT, converted)).isEmpty();
+
+    assertThat(Types.STRING_LIST_DICT.concat(Arrays.asList(converted)));
```

Comment:
I don't quite understand what you are trying to do here. What is the test's assertion?

Reply:
You are right, I added an assertions to check if it is equal to expected.

Reply:
I see, you originally wanted to just check that the thing doesn't crash. I would suggest doing a bit more than that, and splitting out the tests for concat into a new method - let's say 

```java
  @Test
  public void testStringListDict_concat() throws Exception {
    ...
  }
```

Now, what should we test in that method? There are 3 important cases to check:
* concat of an iterable of zero items should return an empty map.
* concat of an iterable containing one map should return a map with the same content (i.e. `assertThat(Types.STRING_LIST_DICT.concat(ImmutableList.of(map)).isEqualTo(map)`) - and obviously should not crash
* concat of an iterable containing two maps should return a map with the same content, and the second item's values overriding the first item's; in other words, if the first map is `{"foo": ["a", "b"], "bar": ["c", "d"]}` and the second map is `{"bar": ["x", "y"], "baz": ["z"]}`, a `concat` of them is expected to produce `{"foo": ["a", "b"], "bar": ["x", "y"], "baz": ["z"]}`